src/routes/health.js

```diff
@@ -0,0 +1,75 @@
+import express from 'express'
+import config from '../../config/index.js'
+import AWS from 'aws-sdk'
+import { createClient } from 'redis'
+
+const router = express.Router()
+
+AWS.config.update({
+  region: config.aws.region,
+  endpoint: config.aws.endpoint,
+  s3ForcePathStyle: config.aws.s3ForcePathStyle || false
+})
+
+router.get('/', async (req, res) => {
+  const toReturn = {
+    name: config.serviceName,
+    environment: config.environment,
+    version: process.env.GIT_COMMIT || 'unknown',
+    maintenance: config.maintenance.serviceUnavailable,
+    dependencies: [
+      {
+        name: 's3-bucket',
+        status: await checkS3Bucket() ? 'ok' : 'unreachable'
+      },
+      {
+        name: 'request-api',
+        status: await checkRequestApi() ? 'ok' : 'unreachable'
+      },
+      {
+        name: 'redis',
+        status: await checkRedis() ? 'ok' : 'unreachable'
+      }
+    ]
+  }
+
+  res.json(toReturn)
```

Comment:
Looks like this always returns a HTTP status code of 200 OK?  It should return 500 with the response body if any of the dependencies are unreachable.